utils/formatters/number.ts

```diff
@@ -33,6 +33,14 @@ export const getDecimalPlaces = (value: WeiSource) => (value.toString().split('.
 
 export const zeroBN = wei(0);
 
+export const truncateNumbers = (value: WeiSource, maxDecimalDigits: number) => {
+	if (value.toString().includes('.')) {
+		const parts = value.toString().split('.');
+		return parts[0] + '.' + parts[1].slice(0, maxDecimalDigits);
+	}
+	return value.toString();
+};
```

Comment:
Couldn't we do this with math instead of relying on string matching for the decimal? I'm wondering if this will break if someone uses `,` for the decimal point?

Something like this:
```
const truncValue = Math.floor(value.mul(10**maxDecimalDigits).toNumber()) / (10**maxDecimalDigits)
return truncValue.toString();
```